proof-libs/fstar/core/Std.Io.Impls.fsti

```diff
@@ -0,0 +1,2 @@
+module Std.Io.Impls
+val placeholder: unit
```

Comment:
Why is this needed?

Reply:
I don't know, we can skip it and add it later if needed

Reply:
Can you try to git blame maybe? so that we can maybe correlate the date and author of this change with something in sandwich for instance?